packages/e2e-tests/tests/deleted-recording.test.ts

```diff
@@ -0,0 +1,13 @@
+import test, { expect } from "@playwright/test";
+
+import { startTest } from "../helpers";
+
+const deletedRecordingId = "94f1729a-3aab-449b-98d6-10865a5e83c5";
```

Comment:
I kept it outside of examples.json because it doesn't really depend on content of recording, just needs to be "deleted" recording.